src/components/category-selection/CategorySelect.module.scss

```diff
@@ -0,0 +1,134 @@
+@use "../../styles/config/mixins" as *;
+
+$gap: 2px;
+
+.categorySelect {
+  @include flex(column);
+}
+
+.label {
+  font-weight: var(--font-weight-medium);
+  color: var(--color-neutral-2);
+  font-size: var(--font-size-md);
+  margin-bottom: var(--spacing-md);
+}
+
+.select {
+  @include flex(row, space-between, center);
+  @include focus;
+
+  background-color: var(--color-neutral-5);
+  color: var(--color-neutral-2);
+  border-radius: var(--border-radius-inner);
+
+  .placeholder {
+    @include text-truncate;
+
+    padding-right: var(--spacing-sm);
+    flex-grow: 1;
+  }
+
+  &.isOpen {
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+    margin-bottom: 0;
+  }
+}
+
+.sideButton,
+.toggleSelect,
+.selectIcon {
+  @include flex(row, center, center);
+
+  box-sizing: content-box;
+
+  color: var(--color-neutral-3);
+
+  svg {
+    max-width: fit-content;
+  }
+}
+
+.selectIcon {
+  padding-left: var(--spacing-lg);
+  padding-right: var(--spacing-md);
+  padding-top: var(--spacing-sm);
+  padding-bottom: var(--spacing-sm);
+}
+
+.sideButton,
+.toggleSelect {
+  border-left: $gap solid var(--color-neutral-4);
+  padding: var(--spacing-sm) var(--spacing-md);
+}
+
+.selectedCategoriesWrapper {
+  width: 100%;
+  max-width: 260px;
+  .selectedCategories {
+    @include flex(row, flex-start, center);
+    gap: 12px;
```

Comment:
Please refrain from using constant numbers and opt for CSS properties instead. This practice enhances maintainability and consistency across the codebase.

Reply:
Updated the gap, not sure about max-width tho. I can try with `vw` instead but it needs to be a concrete size and it's rather specific.